src/app/account/settings/integration/index.tsx

```diff
@@ -0,0 +1,39 @@
+"use client";
+
+import LabledInput from "@/app/components/ui/labled-input";
+import { useToast } from "@/app/components/ui/use-toast";
+import { useSupabase } from "@/app/supabase-provider";
+import { getAllIntegrationsFromSupabase, saveTelegramIntegration } from "@/lib/utils";
+import { useEffect, useState } from "react";
+
+export function Integration() {
+  const { supabase } = useSupabase();
+  const { toast } = useToast();
+  const [data, setData] = useState<object[]>([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      const integration = await getAllIntegrationsFromSupabase(supabase) as [];
+      let filteredData = [];
+      for (const [key, value] of Object.entries(integration[0])) {
+        filteredData.push({name:key,value:value})
+      }
+      console.log(filteredData)
```

Comment:
```suggestion
```

Reply:
Sorry For that ,I forgot to remove console messages used in testing the funtionality.